chart/templates/ingress.yaml

```diff
@@ -1,23 +1,24 @@
 apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
-  name: incident-manager-ui-{{ .Release.Name }}
```

Comment:
I think we might want to gate this if clerk is true, don't deploy kratos or ui, and route to backend, or deploy kratos, ui and route to ui